src/ngraph_utils.cc

```diff
@@ -140,4 +140,24 @@ tf::Status TFTensorShapeToNGraphShape(const tf::TensorShape& tf_shape,
   return tf::Status::OK();
 }
 
+const std::set<tf::DataType>* NGraphDTypes() {
```

Comment:
Since this is a super set of Numeric and Index datatypes - is there a better way to represent this as opposed to duplicating the code?

Reply:
I kinda like it better this way. If we do anything fancy with the initialization we'll need a mutex/is_initialized dance, which seems a bit hairy.